src/main/java/com/aws/greengrass/deployment/DeviceConfiguration.java

```diff
@@ -679,6 +683,23 @@ public Topic getDeploymentPollingFrequencySeconds() {
         return getTopic(DEPLOYMENT_POLLING_FREQUENCY_SECONDS);
     }
 
+    /**
+     * Get s3 endpoint topic.
+     *
+     * @return s3 endpoint topic
+     */
+    public Topic gets3EndpointType() {
+        Topic s3EndpointType = getTopic(S3_ENDPOINT_TYPE).dflt("");
```

Comment:
you can just call `lookup` here I believe. This bit of code needs more clean-up.

Reply:
The `getTopic` function is using `lookup`, I want to reuse the function for looking up the topic, is that ok?